modules/files.nix

```diff
@@ -102,10 +102,10 @@ in
       default = { };
       example = lib.literalExpression ''
         {
-          "ftplugin/nix.lua".text = '''
-            vim.opt.tabstop = 2
-            vim.opt.shiftwidth = 2
-            vim.opt.expandtab = true
+          "after/ftplugin/nix.lua".text = '''
+            vim.bo.tabstop = 2
```

Comment:
While `vim.bo` is the buffer local version of `vim.o`, I believe `vim.opt_local` is the buffer local version of `vim.opt`.

I don't think it makes a difference for primitive values, but I think we normally use `vim.opt` and its siblings in examples because it has better support for lua tables as values.